src/Microsoft.OData.Core/JsonLight/ODataJsonLightCollectionDeserializer.cs

```diff
@@ -72,6 +73,7 @@ internal ODataJsonLightCollectionDeserializer(ODataJsonLightInputContext jsonLig
                 while (this.JsonReader.NodeType == JsonNodeType.Property)
                 {
                     IEdmTypeReference actualItemTypeRef = expectedItemTypeReference;
+                    this.ReadPropertyCustomAnnotationValue = this.ReadCustomInstanceAnnotationValue;

```

Comment:
Haven't understood this assignment. if they are the same why can't we use <code>this.ReadCustonInsanceAnnotationValue</code> in the place of <code>this,ReadPropertyCustomAnnotationValue </code>

Reply:
@ElizabethOkerio `ReadPropertyCustomAnnotationValue` is a `Func` property declared in the base class `ODataJsonLightDeserializer`. The derived classes are required to initialize it with a function that does the actual reading of custom property annotation value. That is what we're doing here.

Reply:
I think i didn't look at their definitions

Reply:
No problem. Happy to clarify